_includes/v19.1/app/insecure/txn-sample.clj

```diff
@@ -0,0 +1,43 @@
+(ns test.test
+  (:require [clojure.java.jdbc :as j]
+            [test.util :as util]))
+
+;; Define the connection parameters to the cluster.
+(def db-spec {:subprotocol "postgresql"
```

Comment:
Here we use `:subprotocol` and friends, but in the other example we use `:dbtype`, `:host`, etc.  They both work but I think we should standardize (IDK which is better, if either)

Reply:
Agreed! I just opted for not touching the insecure examples, but I will update to standardize. Same for the C++ examples. Most of my syntax is based on some API reference or other than I found.